coordinator/src/main/scala/com/socrata/datacoordinator/secondary/SecondaryWatcherConfig.scala

```diff
@@ -16,6 +16,9 @@ class SecondaryWatcherConfig(config: Config, root: String) {
   val watcherId = UUID.fromString(config.getString(k("watcher-id")))
   val claimTimeout = config.getDuration(k("claim-timeout"), MILLISECONDS).longValue.millis
   val maxRetries = config.getInt(k("max-retries"))
+  val maxReplays = try { Some(config.getInt(k("max-replays"))) } catch { case e: Throwable => None }
```

Comment:
Don't catch Throwable, catch whatever is actually thrown (probably `ConfigException.Missing` here)